src/main.rs

```diff
@@ -53,15 +53,26 @@ struct GenerateFlags {
     #[clap(long)]
     no_postprocess: bool,
 
-    /// Include operations in the output that are marked `"x-hidden": true`.
-    #[clap(long)]
-    include_hidden: bool,
+    /// Which operations to include
+    #[clap(long, value_enum, default_value_t=IncludeMode::OnlyPublic)]
+    include_mode: IncludeMode,
 
     /// Write api.ron and types.ron files, as a debugging aid.
     #[clap(long)]
     debug: bool,
 }
 
+#[derive(Copy, Clone, clap::ValueEnum)]
+#[clap(rename_all = "kebab-case")]
+enum IncludeMode {
+    /// Only public options
+    OnlyPublic,
+    /// Both public operations and operations marked with `x-hidden`
+    PublicAndHidden,
```

Comment:
I get it, but `PublicAndHidden` just kinda makes my brain bleed blood tears when I read it.